test/unit/lib/plugins/aws/package/lib/mergeIamTemplates.test.js

```diff
@@ -245,6 +263,98 @@ describe('lib/plugins/aws/package/lib/mergeIamTemplates.test.js', () => {
       });
     });
 
+    describe('Provider properties - modern usage', () => {
```

Comment:
We should rather update `Provider properties` to use just new versions, and introduce `Provider deprecated properties` to test those deprecated.

It'll also open the door for very easy update when we will upgrade the major (we will simply remove `Provider deprecated properties` section)